### ISSUE TITLE
Shall we get revised sails-redis with latest npm redis version 2.4.2

### ISSUE THREAD

**Issue by rasekar2000** [balderdashy/sails-redis#84]
RajRajen:lb4b-api rajrajen$ npm-check -u

npm WARN unmet dependency //lb4b-api/node_modules/sails-redis requires redis@'~0.12.0' but will load
npm WARN unmet dependency //lb4b-api/node_modules/redis,
npm WARN unmet dependency which is version 2.4.2

**Comment by Ryanc1256** [comment]
@rasekar2000 yes we shale, it is been long overdue!

**Comment by mikermcneil** [comment]
@rasekar2000 @Ryanc1256 :+1:

You guys rock!
 ![](http://geology.com/rocks/pictures/shale-colors-380.jpg)

**Comment by Ryanc1256** [comment]
@mikermcneil rock(s) plural :P and ill try to make a PR with my proposed changes for #80 this weekend and ill update all the dependencies, if I get time to do it... super busy lately lol...

**Comment by rasekar2000** [comment]
Thanks.

**Comment by sailsbot** [comment]
Thanks for posting, @rasekar2000.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by rasekar2000** [comment]
Please reopen .. @Ryanc1256

**Comment by Ryanc1256** [comment]
@rasekar2000 woops done :D

**Comment by rasekar2000** [comment]
:thumbsup:

**Comment by johnabrams7** [comment]
@rasekar2000 @Ryanc1256 Hey everyone, thanks for the all the input so far on this one. We're moving all the redis cases to the main [Sails repo](https://github.com/balderdashy/sails/issues) for a closer eye from the Sails team and some added exposure to the community. I see this issue was re-opened and welcome any further developments!
